frontend/packages/erd-core/src/features/gtm/types.ts

```diff
@@ -0,0 +1,3 @@
+export type CommonLogEvent = {
+  appEnv: string
+}
```

Comment:
This doesn't look like it's being used. 👀

Reply:
The implementation of common logs is planned to be addressed in a separate issue, so I have removed it. 🙏
[🔥 remove unused CommonLogEvent type from GTM types](https://github.com/liam-hq/liam/pull/305/commits/7bef593b384e1b244e76b7789c62f7901ba7d395)

Reply:
Oh! OK, Please continue after this!